boilerplate/openshift/golang-osd-operator/golangci.yml

```diff
@@ -18,6 +18,7 @@ linters:
   enable:
   - deadcode
   - errcheck
+  - gosec
```

Comment:
We should look at adding rules incrementally to address Eric's concern about all consuming repos failing to pass the lint PR checks once they consume this change. https://golangci-lint.run/usage/linters/#gosec

Reply:
Maybe just G101-G112 for now? https://github.com/securego/gosec#available-rules

Reply:
Sure, feel free to recreate the PR.

Also I know @Tessg22 is doing the painstakingly hard process of checking per-repo.

Please collab with her if possible on this